code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1090,7 +1092,7 @@
 						H.update_inv_glasses()
 
 			if("chest")	//Easier to score a stun but lasts less time
-				if(H.stat == CONSCIOUS && I.force && prob(I.force + 10))
+				if(H.stat == CONSCIOUS && knockout && prob(I.knockout + 10))
```

Comment:
`if(H.stat == CONSCIOUS && I.knockout && prob(I.knockout + 10))`

Please read what I wrote to what you currently have IN IT'S ENTIRETY